pkg/odo/cli/env/env_test.go

```diff
@@ -45,8 +46,15 @@ func TestPrintSupportedParameters(t *testing.T) {
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
 			got := strings.TrimSpace(printSupportedParameters(tt.supportedParameters))
-			if !reflect.DeepEqual(got, tt.want) {
-				t.Errorf("Got %s, want %s", got, tt.want)
+
+			gotStrings := strings.Split(got, "\n")
+			wantStrings := strings.Split(tt.want, "\n")
+
+			sort.Strings(gotStrings)
```

Comment:
That works, too. 👍 
And I was actually thinking that maybe that's what `printSupportedParameters` should do (sort by parameter name) as well to get consistent output.

Reply:
I thought about that. But I didn't want to modify the actual function. So left it unchanged for now.